{
  "Entries": [
    {
      "RequestUri": "https://api.loganalytics.io/v1/workspaces/97fa26b5-39ef-45ce-8f17-e4359d0d2730/query",
      "RequestMethod": "POST",
      "RequestHeaders": {
        "Accept": "application/json",
        "Authorization": "Sanitized",
        "Content-Length": "138",
        "Content-Type": "application/json",
        "traceparent": "00-b72ca2f843aa0b48a29000065571a1f4-a5ae91ab333a6049-00",
        "User-Agent": [
          "azsdk-net-Monitor.Query/1.0.0-alpha.20210914.1",
          "(.NET 5.0.10; Microsoft Windows 10.0.19043)"
        ],
        "x-ms-client-request-id": "25b5de3f5eba54423babdd99d1c03f40",
        "x-ms-return-client-request-id": "true"
      },
      "RequestBody": {
        "query": "set truncationmaxrecords=1; datatable (s: string) [\u0027a\u0027, \u0027b\u0027]",
        "timespan": "2021-09-06T00:00:00.0000000Z/P7D"
      },
      "StatusCode": 200,
      "ResponseHeaders": {
        "Access-Control-Allow-Origin": "*",
        "Access-Control-Expose-Headers": "Retry-After,Age,WWW-Authenticate,x-resource-identities,x-ms-status-location",
        "Age": "0",
        "Connection": "keep-alive",
        "Content-Length": "509",
        "Content-Type": "application/json; charset=utf-8",
        "Date": "Wed, 15 Sep 2021 00:03:10 GMT",
        "Strict-Transport-Security": "max-age=15724800; includeSubDomains",
        "Vary": [
          "Accept-Encoding",
          "Accept-Encoding"
        ],
        "Via": "1.1 draft-oms-5cbfc96d4-ncmt4",
        "X-Content-Type-Options": "nosniff"
      },
      "ResponseBody": {
        "tables": [
          {
            "name": "PrimaryResult",
            "columns": [
              {
                "name": "s",
                "type": "string"
              }
            ],
            "rows": [
              [
                "a"
              ]
            ]
          }
        ],
        "error": {
          "code": "PartialError",
          "message": "There were some errors when processing your query.",
          "details": [
            {
              "code": "EngineError",
              "message": "Something went wrong processing your query on the server.",
              "innererror": {
                "code": "-2133196797",
                "message": "Query result set has exceeded the internal record count limit 1 (E_QUERY_RESULT_SET_TOO_LARGE; see https://aka.ms/kustoquerylimits)",
                "severity": 2,
                "severityName": "Error"
              }
            }
          ]
        }
      }
    }
  ],
  "Variables": {
    "DateTimeOffsetNow": "2021-09-14T17:03:10.3734827-07:00",
    "LOGS_ENDPOINT": "https://api.loganalytics.io",
    "RandomSeed": "1967895409",
    "WORKSPACE_ID": "97fa26b5-39ef-45ce-8f17-e4359d0d2730"
  }
}